### ISSUE TITLE
Execute sql batch statements

### ISSUE THREAD

**Issue by slevy85** [jepiqueau/jeep#3]
Hi thank you for your efforts to develop a good sqlite plugin for capacitor.

I need to execute many sql inserts in batch and i have values to bind to them.
Is it possible to improve the execute function so it can have an API similar to this ?
```
execute({
statements:[
     `insert into users(id,name) values(?1,?2)`,
     `insert into post(title) values(?1)`
],
values: [
     [1234, "name1"],
     ["<div dir=\"ltr\">Title</div>"],
]
})
```

Thank you for your help.

**Comment by slevy85** [comment]
In fact, I am thinking again and I think the current implementation that takes a singl string would be enough for me.

**Comment by jepiqueau** [comment]
@slevy85 Thanks for using the capacitor-sqlite plugin. As you said, the run command for single statement address this.The separator between values is a comma not ;
you can still do that even if i haven't try yet
```ts
      const row: Array<Array<Any>> = [ ["Whiteley","Whiteley.com",30], ["Jones","Jones.com",44]]
      const sqlcmd: String = `
      BEGIN TRANSACTION;
      DELETE FROM users;
      INSERT INTO users (name,email,age) VALUES (${row[0]});
      INSERT INTO users (name,email,age) VALUES (${row[1]});
      COMMIT TRANSACTION;
      `;
I think it should work
      retExe = await this._sqlite.execute({statements:sqlcmd});

```

**Comment by jepiqueau** [comment]
@slevy85 Come back to you, after testing, it should be:
```ts
              const row: Array<Array<any>> = [["Whiteley","Whiteley.com",30],["Jones","Jones.com",44]];
              const sqlcmd: string = `
              BEGIN TRANSACTION;
              DELETE FROM users;
              INSERT INTO users (name,email,age) VALUES ("${row[0][0]}","${row[0][1]}",${row[0][2]});
              INSERT INTO users (name,email,age) VALUES ("${row[1][0]}","${row[1][1]}",${row[1][2]});
              COMMIT TRANSACTION;
              `;
``` 
my first answer was too quick.
Hope this close the issue

**Comment by slevy85** [comment]
What if the values contains a `;` `["Whiteley","Whiteley;com",30]`? I think it will fail on android because the string is split by `;` : https://github.com/jepiqueau/jeep/blob/fa0853f50c3113f19d03f28d99bf972ba5f7964e/capacitor/plugins-library/android/src/main/java/com/jeep/plugins/capacitor/CapacitorSQLite.java#L124

**Comment by jepiqueau** [comment]
@slevy85  this issue has been addressed in  release 0.0.3-2 of the capacitor-sqlite
